Dockerfile

```diff
@@ -1,33 +1,18 @@
-FROM node:18 AS build
-
+FROM node:18-alpine AS build
 WORKDIR /app
-
-
-COPY package.json ./
-COPY package-lock.json ./
-
-# Install dependencies
-RUN npm install --frozen-lockfile
-
+COPY package*.json ./
+RUN npm ci
```

Comment:
Why did you decided to use `npm ci`

Reply:
because it’s faster and makes sure everything installs exactly as in package-lock.json. Plus, it's quicker since it skips some steps like package updates.